dist/common/scripts/scylla_util.py

```diff
@@ -137,10 +141,9 @@ def is_gce_instance():
                     return True
         return False
 
-    def __instance_metadata(self, path):
-        """query GCP metadata server, recursively!"""
-        return curl("http://169.254.169.254/computeMetadata/v1/instance/" + path+"?recursive=true", headers={"Metadata-Flavor": "Google"})
-        #169.254.169.254 is metadata.google.internal
+    def __instance_metadata(self, path, recursive=False):
+        return curl(self.META_DATA_BASE_URL + path + "?recursive=%s" % str(recursive).lower(),
```

Comment:
Why did you remove the comment?

Reply:
after the change, the information in the comment is not useful here